x-pack/docs/en/settings/security-settings.asciidoc

```diff
@@ -748,191 +768,231 @@ As per `attribute_patterns.principal`, but for the _dn_ property.
 
 `nameid_format`::
 The NameID format that should be requested when asking the IdP to authenticate
-the current user.
-Defaults to `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`
+the current user. Defaults to requesting _transient_ names 
+(`urn:oasis:names:tc:SAML:2.0:nameid-format:transient`). 
+
+`nameid.allow_create`:: The value of the `AllowCreate` attribute of the 
+`NameIdPolicy` element in an authentication request. Defaults to `false`. 
+
+`nameid.sp_qualifier`:: The value of the `SPNameQualifier` attribute of the 
+`NameIdPolicy` element in an authentication request. The default is to not 
+include the `SPNameQualifier` attribute.
 
 `force_authn`::
-Whether to set the `ForceAuthn` attribute when requesting that the IdP
-authenticate the current user.
-Defaults to `false`.
+Specifies whether to set the `ForceAuthn` attribute when requesting that the IdP
+authenticate the current user. If set to `true`, the IdP is required to freshly 
+establish the user’s identity, irrespective of any exiting 
+//TBD: Should this be "existing"?
```

Comment:
Yes, "existing"
I think it might also be helpful to use something that reads a little more smoothly than "freshly establish"

Reply:
Thanks, @tvernum!  I changed the verb to "verify", but if that's not what you had in mind, let me know.